lib/basedriver/commands/find.js

```diff
@@ -1,174 +0,0 @@
-import log from '../logger';
-import { logger } from 'appium-support';
-import _ from 'lodash';
-import { errors } from '../../..';
-
-
-const commands = {}, helpers = {}, extensions = {};
-
-const CUSTOM_STRATEGY = '-custom';
-
-// Override the following function for your own driver, and the rest is taken
-// care of!
-
-// helpers.findElOrEls = async function (strategy, selector, mult, context) {}
-//   strategy: locator strategy
-//   selector: the actual selector for finding an element
-//   mult: multiple elements or just one?
-//   context: finding an element from the root context? or starting from another element
-//
-// Returns an object which adheres to the way the JSON Wire Protocol represents elements:
-// { ELEMENT: # }    eg: { ELEMENT: 3 }  or { ELEMENT: 1.023 }
-
-helpers.findElOrElsWithProcessing = async function findElOrElsWithProcessing (strategy, selector, mult, context) {
-  this.validateLocatorStrategy(strategy);
-  try {
-    return await this.findElOrEls(strategy, selector, mult, context);
-  } catch (err) {
-    if (this.opts.printPageSourceOnFindFailure) {
-      const src = await this.getPageSource();
-      log.debug(`Error finding element${mult ? 's' : ''}: ${err.message}`);
-      log.debug(`Page source requested through 'printPageSourceOnFindFailure':`);
-      log.debug(src);
-    }
-    // still want the error to occur
-    throw err;
-  }
-};
-
-commands.findElement = async function findElement (strategy, selector) {
-  if (strategy === CUSTOM_STRATEGY) {
-    return await this.findByCustom(selector, false);
-  }
-
-  return await this.findElOrElsWithProcessing(strategy, selector, false);
-};
-
-commands.findElements = async function findElements (strategy, selector) {
-  if (strategy === CUSTOM_STRATEGY) {
-    return await this.findByCustom(selector, true);
-  }
-
-  return await this.findElOrElsWithProcessing(strategy, selector, true);
-};
-
-commands.findElementFromElement = async function findElementFromElement (strategy, selector, elementId) {
-  return await this.findElOrElsWithProcessing(strategy, selector, false, elementId);
-};
-
-commands.findElementsFromElement = async function findElementsFromElement (strategy, selector, elementId) {
-  return await this.findElOrElsWithProcessing(strategy, selector, true, elementId);
-};
-
-/**
- * Find an element using a custom plugin specified by the customFindModules cap.
- *
- * @param {string} selector - the selector which the plugin will use to find
- * elements
- * @param {boolean} multiple - whether we want one element or multiple
- *
- * @returns {WebElement} - WebDriver element or list of elements
- */
-commands.findByCustom = async function findByCustom (selector, multiple) {
```

Comment:
we don't want to delete the whole file. only this function and ones immediately related to it. all the other find element stuff we want to keep in here.